src/assets/locales/fr.json

```diff
@@ -385,7 +429,7 @@
   "preferences-screen.refresh-now": "Rafraichir maintenant",
```

Comment:
"Rafraîchir maintenant"

Reply:
No I'm using the 1990 reformed orthograph, without accent. :) 
https://www.larousse.fr/dictionnaires/francais/rafra%C3%AEchir/66192

Reply:
Haha feeling old suddenly ;)